### ISSUE TITLE
Missing Dependency @juggle/resize-observer

### ISSUE THREAD

**Issue by drumfreak** [qwefgh90/ng-terminal#62]
When attempting to build angular 12:

```
Error: The target entry-point "ng-terminal" has missing dependencies:
 - @juggle/resize-observer
```

Fix with:
```
 npm i --save @juggle/resize-observer
```

**Comment by qwefgh90** [comment]
@drumfreak Right. it should be included in *dependencies* not in *peerDependencies*.

**Comment by qwefgh90** [comment]
It's resolved by #63.

**Comment by drumfreak** [comment]
Thank you!
